README.md

```diff
@@ -1 +1,295 @@
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard_project] <a name="about-project"></a>
+
+## 🛠 Built With <a name="VSCode, Git and Github"></a>
+
+### Tech Stack <a name="Html, Webpack, Gitflow, Linters, Css and JavaScript"></a>
+
+<details>
+  <summary>Client side is the language used to develop the project </summary>
+  <ul>
+    <li><a href="https://html.com/">Html</a></li>
+    <li><a href="https://javascript.com/">JavaScript</a></li>
+    <li><a href="https://csstricks.com/">Css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server helps to host the project on a localhost </summary>
+  <ul>
+    <li><a href="https://webpack.js.org/guides/development/#using-webpack-dev-server">Webpack-dev-server</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">Database</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Submit feature]**
+- **[Refresh feature]**
+- **[Recent scores feature]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://lancelot-so.github.io/Leaderboard_project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+- Browser
+- git
+- code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Lancelot-SO/Leaderboard_project.git
+```
+```
+    git@github.com:Lancelot-SO/Leaderboard_project.git 
+```
+- Navigate to the To-do with this command 
+```
+    cd Leaderboard_project
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
```

Comment:
- [ ] Kindly update these sections of your README as they are mostly dummy data from the template. It can also be helpful for potential visitors to see how to set up and run the project locally.